endo_api/settings_dev.py

```
@@ -0,0 +1,46 @@
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)